call.sh

```diff
@@ -1,13 +1,26 @@
 #!/bin/bash
-declare -a Array
-curl https://www.cnn.com/ | grep -Eo '*\/[0-9]{4}\/[0-9]{2}\/[0-9]{2}\/[a-z/-]+.html' | awk '$0="https://www.cnn.com"$0' | mapfile -t Array
+depth=1
+input="/home/matthew/MLD/urls.txt"
```

Comment:
Should not be hardcoded. try $HOME/MLD/urls.txt

Reply:
update moved to optimization branch made irrelevant.